### DISCUSSION TITLE
Confirmed CHIA not received in the wallet.

### DISCUSSION THREAD

**Discussion by AHP837** [Chia-Network/chia-blockchain#5578] [Support]
Hello,
There is an incoming XCH of 0.745, but unfortunately, I can not see it in my wallet.
Can you please let me what I should do to see it in the wallet?
Regards.

**Comment by AHP837** [comment]
![Screenshot_1](https://user-images.githubusercontent.com/84499787/118950811-d2d06500-b95a-11eb-8625-c929c926588f.png)

**Reply by AHP837** [reply]
This a screenshot.

**Reply by steven9046** [reply]
I encountered the same problem.

**Reply by AHP837** [reply]
Could u fix it?

**Comment by BrandtH22** [comment]
@AHP837 , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
